plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -128,11 +134,13 @@ test.describe( 'Add New Variable Product Page', () => {
 				.first()
 				.type( 'val1 | val2' );
 		}
-		await page.click( 'text=Save attributes' );
-		// wait for the attributes to be saved
+
+		await page.getByRole( 'button', { name: 'Save attributes'} ).click( { clickCount: 3 });
```

Comment:
There is something really off with the Save attributes button.  It doesn't ever appear to become active.